src/player.js

```diff
@@ -1725,6 +1863,16 @@ export default class Player extends FakeEventTarget {
     }
   }
 
+  /**
+   * Update the floating active state.
+   * @param {boolean} isFloating - is player floating.
+   * @returns {void}
+   * @public
+   */
+  changeFloatingState(isFloating: boolean): void {
```

Comment:
`toggleFloatingState` ? or just `floatingState` setter?

Reply:
not relevant anymore